blaze/expr/datetime.py

```diff
@@ -119,6 +129,27 @@ def utcfromtimestamp(expr):
     return UTCFromTimestamp(expr)
 
 
+class nanosecond(DateTime): _dtype = datashape.int64
```

Comment:
True. I had simply cloned from the existing classes above. I assume I should fix all of those as well while I'm at it ?